.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -23,8 +23,24 @@ A clear and concise description of what you expected to happen.
 If applicable, add screenshots to help explain your problem.
 
 **Configuration (please complete the following information):**
- - OS (e.g. Windows):
- - Nu version (you can use the `version` command to find out): 
- - Optional features (if any):
 
-Add any other context about the problem here.
+Run `version` command and paste the output to show OS, features, etc.
+
+```
+ version            │ 0.24.0
+ branch             │ main
+ short_commit       │ 21882e5d
+ commit_hash        │ 21882e5d19eab7b8ed340ad38396234a15db7b35
+ commit_date        │ 2020-12-08T05:10:51+00:00
+ build_os           │ macos-x86_64
+ rust_version       │ rustc 1.48.0 (7eac88abb 2020-11-16)
+ rust_channel       │ stable-x86_64-apple-darwin (default)
+ cargo_version      │ cargo 1.48.0 (65cbdd2dc 2020-10-14)
+ pkg_version        │ 0.23.0
+ build_time         │ 2020-12-08 08:35:51
+ build_rust_channel │ debug
+ features           │ ctrlc, default, directories, dirs, ...
```

Comment:
I think by default `version` will print the table the other direction, which might be hard to copy/paste.

@fdncred - any thoughts?

Reply:
Ya, I have a custom command that I use now `def nuver [] { version | pivot key value }`

Reply:
Maybe we should tell them to run `version | pivot key value`

Reply:
Great point, updated to use `version | pivot`.